### PR TITLE
enhancement/sulphur_pd_allegations

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: clean/sulphur_pd_cprr.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import sys
+
+sys.path.append("../")
```
Comment (pckhoi) [inline clean/sulphur_pd_cprr.py:3]: You can remove `sys.path.append('../')` for all scripts now that we have Deba.

Comment (ayyubibrahimi) [reply clean/sulphur_pd_cprr.py:3]: Thanks

File: data/datavalid.yml [hunk 2]
```diff
@@ -164,6 +164,7 @@ schemas:
         description: >
           small departments frequently turnover internal investigations to the sheriff's office
       - name: incident_location
+      - name: disposition_desc
```
[the remark this answers is not in the document]
Comment (ayyubibrahimi) [reply data/datavalid.yml:167]: Thanks

**Comment by ayyubibrahimi** [comment]
Ready for review. Thanks!

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**